README.md

```diff
@@ -22,15 +22,24 @@ Similar to the [Rust community's](https://this-week-in-rust.org/), we want to ha
 - Tell active contributors when new versions or tools are available.
 - Attribute developers who have contributed to open-source software related to IPFS.
 
+## Contribute
+
+The Weekly takes a _lot of work_. This work can best be done by being distributed. If you know of anything cool that happens in a given week, anything that other people might be excited about, add it to the weekly by adding a comment about it to the PR or Issue for that week's weekly.
```

Comment:
I wish we could create a perma link that linked to the latest, to reduce friction to submitting content as much as possible.